runner/llamarunner/runner.go

```diff
@@ -474,7 +474,7 @@ func (s *Server) processBatch(tokenBatch *llama.Batch, embedBatch *llama.Batch)
 			}
 
 			seq.embedding <- embed
-			s.removeSequence(i, "")
+			s.removeSequence(i, llm.DoneReasonComplete)
```

Comment:
I think this naturally would be `DoneReasonStop` - the `embed` endpoint doesn't return a reason so it doesn't have any effect but it still seems right. This is the only use of the somewhat ambiguous `DoneReasonComplete`, we would reduce the number of states as well.